data/views/node.2.0.json

```diff
@@ -13,18 +21,24 @@
 <% } else  {%>
 "tags": [],
 <% } %>
-<% if (hasOwnProperty("obmSettings")) { %>
-"obmSettings": <%- JSON.stringify(obmSettings) %>,
-<% } else  {%>
-"obmSettings": [],
+<% if (obms == null) { %>    
```

Comment:
whether there is a case the `obms` is not in the rendering context? If so, this line will fail. Actually I just fixed a lot such kind of issue for install-os auto-install scripts, see https://github.com/RackHD/on-http/pull/347